UnityProject/Assets/Scripts/Chat/Chat.cs

```diff
@@ -68,7 +68,9 @@ public static void AddChatMsgToChat(ConnectedPlayer sentByPlayer, string message
 		{
 			processedMessage = ProcessMessage(sentByPlayer, message);
 
-			if (player.mind.occupation.JobType == JobType.MIME && player.mind.IsMiming
+			var occupation = player.mind.occupation;
+
+			if (occupation != null && !player.mind.IsGhosting && occupation.JobType == JobType.MIME && player.mind.IsMiming
```

Comment:
actually this jobtype check is redundant, IsMiming should be enough
```suggestion
			if (player.mind.IsMiming
```

Reply:
Cleaned it up but added !player.IsDeadOrGhost